### PR TITLE
Add new funnels

### PR DESCRIPTION
**Pull request by swillis**
Adding 5 new 'funnel' pages to the splash pages repo (as part of the activation theme).

**DD funnel**
This page targets existing users of Direct Debit, and tells them of the merits of switching to GoCardless.

dd funnel

**Non DD funnel**
Aimed at merchants who do not currently use Direct Debit to collect payments. It highlights the benefits of using DD with GoCardless.

non dd funnel

**SMB funnel**
Targets small-to-mid sized businesses (duh).

smb funnel

**Developer funnel**
To help developers understand how they can get started with our API.

developers

**Integrate funnel**
To help developers understand how they can integrate with our API.

integrate

### CODE REVIEW

**Comment by swillis** [comment]
@jamesshedden please could you give this a review when you get a moment? Muchas gracias 😘

File: app/css/components/product-grid.scss [hunk 1]
```diff
@@ -12,3 +12,7 @@
 .product-grid__section--first {
   border-right: 1px solid $gray;
 }
+
+.product-grid__section--second {
```
Comment (jamesshedden) [inline app/css/components/product-grid.scss]: @swillis Anyway to refactor this using `nth-child`? Let's chat 💭

File: app/css/components/security-benefit.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.security-margin {
```
Comment (jamesshedden) [inline app/css/components/security-benefit.scss]: Rename class to match component `.security-margin` -> `.security-benefit`

File: app/css/components/slider.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.slider__nav {
```
Comment (jamesshedden) [inline app/css/components/slider.scss]: Any reason why `.slider__nav` as top level in `slider` component file without a parent `.slider` class? 

Can use `slider` as parent eg `.slider`, `.slider__next`, etc

File: app/css/components/svg-icon.scss [hunk 4]
```diff
@@ -4,6 +4,16 @@
   position: relative;
 }
 
+.svg-icon--customisable-integration {
```
Comment (jamesshedden) [inline app/css/components/svg-icon.scss]: `.svg-icon--large`?

File: app/css/components/svg-icon.scss [hunk 5]
```diff
@@ -25,6 +35,16 @@
   max-width: 76px;
 }
 
+.svg-icon__image--customisable-integration {
```
Comment (jamesshedden) [inline app/css/components/svg-icon.scss]: `.svg-icon__image--large`?

File: app/css/components/useful-resource.scss [hunk 6]
```diff
@@ -0,0 +1,8 @@
+.useful-resource-icon {
```
Comment (jamesshedden) [inline app/css/components/useful-resource.scss]: `.useful-resource__icon` & `.useful-resource__title`? Class naming doesn't match component file naming at the mo 😢

File: app/css/main.scss [hunk 7]
```diff
@@ -31,10 +32,12 @@
 @import "components/header-logo";
 @import "components/highlight";
 @import "components/highlight-text";
+@import "components/hljs";
```
Comment (jamesshedden) [inline app/css/main.scss]: Not a component so might be better in one of the other folders? eg `shared` — so maybe move this to `shared` folder, then update here eg `@import "shared/hjls"` (and move location of import to be with other imports from `shared`)

File: app/messages/en.js [hunk 8]
```diff
@@ -228,6 +229,26 @@ export default {
     nav_title: 'Payout tracking',
     description: '',
   },
+  non_dd_funnel: {
```
Comment (jamesshedden) [inline app/messages/en.js]: Can we rename these, and any other things in the app that have similar naming, to something more strictly related to the page contents, eg `newToDirectDebit`, `existingDirectDebitUser`, `smallMediumBusinesses`, `developerFeatures`, `apiFeatures` (suggestions 😄) — be nice to remove references to `funnel` from these & treat them like other pages I think!

File: app/pages/funnels/developer-funnel/developer-funnel.js [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+import Page from '../../../components/page/page';
+import DeveloperFunnelEn from './developer-funnel.en';
+
+export default class DeveloperFunnel extends React.Component {
+  displayName = 'DeveloperFunnel'
+
+  render() {
+    return (
+      <Page isInverted>
+        <DeveloperFunnelEn />
```
Comment (jamesshedden) [inline app/pages/funnels/developer-funnel/developer-funnel.js]: Any reason to separate out this template into another component? Normal pattern is to only do this when there are multiple translated components eg `DeveloperFunnelEn`, `DeveloperFunnelFr` — quite an easy refactor so don't think there's much benefit to thinking ahead with that right now, so contents of `developer-funnel.en.js` can just live in this file, inside the `<Page>` component

File: app/pages/funnels/integrate-funnel/integrate-funnel.js [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+import Page from '../../../components/page/page';
+import IntegrateFunnelEn from './integrate-funnel.en';
+
+export default class IntegrateFunnel extends React.Component {
+  displayName = 'IntegrateFunnel'
+
+  render() {
+    return (
+      <Page isInverted>
+        <IntegrateFunnelEn />
```
Comment (jamesshedden) [inline app/pages/funnels/integrate-funnel/integrate-funnel.js]: Same comment re: `DeveloperFunnelEn`

File: app/pages/funnels/non-dd-funnel/non-dd-funnel.js [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+import Page from '../../../components/page/page';
+import NonDDFunnelEn from './non-dd-funnel.en';
+
+export default class NonDDFunnel extends React.Component {
+  displayName = 'NonDDFunnel'
+
+  render() {
+    return (
+      <Page isInverted>
+        <NonDDFunnelEn />
```
Comment (jamesshedden) [inline app/pages/funnels/non-dd-funnel/non-dd-funnel.js]: Same comment re: `DeveloperFunnelEn`

File: app/pages/home/home.en.js [hunk 12]
```diff
@@ -1,212 +1,43 @@
...
-import CustomersRedIcon from '../../icons/svg/customers-red';
-import SmallBusinessYellowIcon from '../../icons/svg/small-business-yellow';
-import EnterpriseGreenIcon from '../../icons/svg/enterprise-green';
+//
+// import AutomatedYellowIcon from '../../icons/svg/automated-yellow';
+// import SaveMoneyGreenIcon from '../../icons/svg/save-money-green';
+// import LocationGreenIcon from '../../icons/svg/location-green';
+// import CustomersRedIcon from '../../icons/svg/customers-red';
+// import SmallBusinessYellowIcon from '../../icons/svg/small-business-yellow';
+// import EnterpriseGreenIcon from '../../icons/svg/enterprise-green';
```
Comment (jamesshedden) [inline app/pages/home/home.en.js]: ✂️

File: app/pages/funnels/smb-funnel/smb-funnel.js [hunk 13]
```diff
@@ -0,0 +1,15 @@
...
+import Page from '../../../components/page/page';
+import SMBFunnelEn from './smb-funnel.en';
+
+export default class SMBFunnel extends React.Component {
+  displayName = 'SMBFunnel'
+
+  render() {
+    return (
+      <Page isInverted>
+        <SMBFunnelEn />
```
Comment (jamesshedden) [inline app/pages/funnels/smb-funnel/smb-funnel.js]: Same comment re: `DeveloperFunnelEn`

File: app/router/routes.js [hunk 14]
```diff
@@ -353,6 +359,36 @@ export const config = Immutable.fromJS([
       },
     },
   ],
+  [NonDDFunnelEn, { name: 'non_dd_funnel'}, {
+      'en-GB': {
+        path: '/non-dd-funnel',
```
Comment (jamesshedden) [inline app/router/routes.js]: Guessing we will update these urls 💻

File: app/css/components/product-grid.scss (hunk 1 above)
Comment (swillis) [inline app/css/components/product-grid.scss]: 👍

File: app/css/components/security-benefit.scss (hunk 2 above)
Comment (swillis) [inline app/css/components/security-benefit.scss]: 👍

File: app/css/components/slider.scss (hunk 3 above)
Comment (swillis) [inline app/css/components/slider.scss]: 👍

File: app/css/components/svg-icon.scss (hunk 4 above)
Comment (swillis) [inline app/css/components/svg-icon.scss]: 👍

File: app/css/components/svg-icon.scss (hunk 5 above)
Comment (swillis) [inline app/css/components/svg-icon.scss]: 👍

File: app/css/main.scss (hunk 7 above)
Comment (swillis) [inline app/css/main.scss]: 👍

File: app/pages/small-medium-businesses/small-medium-businesses.js [hunk 15]
```diff
@@ -0,0 +1,345 @@
...
+                <div className='u-pull-start u-inline-block u-size-2of3 u-margin-Lm'>
+                  <h3 className='u-text-heading u-color-dark-gray u-text-light u-text-no-smoothing u-text-m'>
+                    Improve cashflow
+                  </h3>
+
+                  <p className='u-color-dark-gray u-margin-Txs'>
+                    Get paid on time, every time. No more chasing late payments or bad debt.
+                  </p>
+                </div>
+              </div>
```
Comment (jamesshedden) [inline app/pages/small-medium-businesses/small-medium-businesses.js]: @swillis Can we do the same centering trick on these sections? Think might be suffering from same issue we were looking at with the other similar component — let's chat thru 💭

File: app/css/components/code-examples.scss [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+
+.code-examples__tabs-code {
+    background-color: inherit;
+    display: block;
+    padding: 0 15px 0 0;
+    border: none
+}
+
+.code-examples__tabs-code:first-child {
+    padding: 0 15px
```
Comment (harrison) [inline app/css/components/code-examples.scss]: indentation 🗡

File: app/css/shared/hljs.scss [hunk 17]
```diff
@@ -0,0 +1,98 @@
+.hljs {
+    background-color: #FBFBFB;
+    color: #586e75;
+    word-wrap: break-word;
+    overflow: visible;
+    display: block;
+    line-height: 24.5px;
+    font-family: "Source Code Pro", Consolas, Monaco, Menlo, "Courier New", monospace;
+    padding: 1.45em;
```
Comment (harrison) [inline app/css/shared/hljs.scss]: hljs used anywhere?

**Pull request merged**